hbase-client/src/test/java/org/apache/hadoop/hbase/client/TestTableDescriptorBuilder.java

```diff
@@ -365,8 +365,9 @@ public void testStringCustomizedValues() throws HBaseException {
       .setMemStoreFlushSize("256MB").setErasureCodingPolicy("RS-6-3-1024k").build();
     assertEquals(
       "'testStringCustomizedValues', " + "{TABLE_ATTRIBUTES => {DURABILITY => 'ASYNC_WAL', "
-        + "ERASURE_CODING_POLICY => 'RS-6-3-1024k', MAX_FILESIZE => '10737942528 B (10GB 512KB)', "
-        + "MEMSTORE_FLUSHSIZE => '268435456 B (256MB)'}}, "
+        + "MAX_FILESIZE => '10737942528 B (10GB 512KB)', "
+        + "MEMSTORE_FLUSHSIZE => '268435456 B (256MB)', "
+        + "METADATA => {'ERASURE_CODING_POLICY' => 'RS-6-3-1024k'}}}, "
```

Comment:
Oh ok, this is why we want to keep it in the defaults map. This setting should not be considered metadata. I couldn't remember this when I filled the jira. 

Thank you for the PR. Now that I see this, I think we should keep it as it was. Maybe you can add a comment where it is added to the defaults map?

Reply:
Thanks @bbeaudreault. Reverted the change and added a comment for future reference. Please Review